### PR TITLE
Tagged CHOICEs must always be explicit.

### PR DESCRIPTION
**Pull request by james-ward**
Current code incorrectly allows implicit CHOICE tags.

### CODE REVIEW

File: asn1ate/sema.py [hunk 1]
```diff
@@ -25,7 +25,6 @@
 
 from asn1ate import parser
 
-
```
Comment (kimgr) [inline asn1ate/sema.py]: pylint wants two blank lines between imports and first code entity, can you please add that back?

**Comment by kimgr** [comment]
This looks obviously good! I've tried to find this rule in the X.680 spec, though, and I can't... Can you cite a source, and add a comment referring to it in the code?

**Comment by james-ward** [comment]
This commit isn't completely correct - it needs the functionality in the multi-module patch in order to be done properly.
At the moment the choice type isn't correctly identified as the root class in other referenced modules. With the addition of the 'referenced_modules' argument in the other patch, adding this when resolving the tag implicity is trivial - but I will have to wait until we are happy with the other patch and it is merged.

**Comment by kimgr** [comment]
Merged, thanks!

**Pull request merged**